src/drivers/BtcDotCom.driver.js

```diff
@@ -5,13 +5,14 @@ export default class BtcDotCom extends Driver {
   constructor (params) {
     super()
 
-    const { version, token } = Object.assign({
+    const { version, token, network } = Object.assign({
```

Comment:
Put this changes in readme please. 
I guess now for this driver your should pass `chain` for mainnet, and `tchain` for testnet - it's kinda not obvious. This why it should be in readme